cmd/skuba/addon/upgrade.go

```diff
@@ -46,7 +48,11 @@ func newUpgradePlanCmd() *cobra.Command {
 		Use:   "plan",
 		Short: "Plan addon upgrade",
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := addons.Plan(); err != nil {
+			clientSet, err := kubernetes.GetAdminClientSet()
+			if err != nil {
+				klog.Errorf("Can not retrieve the ClientSet from kubernetes: %v", err)
+			}
```

Comment:
Make it aligns https://github.com/SUSE/skuba/blob/master/cmd/skuba/node/remove.go#L41-L44 [1]

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md
I would rather to change remove.go to make it consistent.